libs/shared/registration/src/lib/shared-registration.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import {
+  Box,
+  Grid,
+  TextField,
+  Typography,
+  InputAdornment,
+  Button,
+} from '@material-ui/core';
+import Person from '@material-ui/icons/Person';
+import LockIcon from '@material-ui/icons/Lock';
+import PhoneIcon from '@material-ui/icons/Phone';
+import './shared-registration.scss';
+
+interface RegistrationInterface {
+  email: string;
+  password: string;
+  phone?: string;
+}
+
+interface RegistrationFormData{
+  email: string;
+  password: string;
+  repeatPassword: string;
+  phone?: string;
+}
+
+interface SharedRegistrationProps {
```

Comment:
add export on front of it, so you could import it inside shared-registration.stories.tsx: 
![Screenshot 2020-07-27 at 19 10 30](https://user-images.githubusercontent.com/24617345/88576161-e82b7a00-d03c-11ea-8c03-a4727c816be5.png)